templates/accounts/profile.html

```diff
@@ -24,13 +24,29 @@ <h1 class="settings-title">Settings</h1>
             <div class="panel-body settings-block">
                 <form method="post">
                     {% csrf_token %}
-                    <h2>Monthly Reports</h2>
+                    <h2>Select Your Report Type:</h2>
                     <label>
+                        <!-- Radio button to select daily reports-->
                         <input
                             name="reports_allowed"
-                            type="checkbox"
-                            {% if profile.reports_allowed %} checked {% endif %}>
-                        Each month send me a summary of my checks
+                            value="daily"
+                            type="radio"
+                            {% if profile.reports_allowed and reports_allowed == "daily" %} checked {% endif %}>
```

Comment:
There is still a tad bit of repition here. You should try and keep the html dry also and stay away from if else statements